### ISSUE TITLE
top/bottom links are not consistent

### ISSUE THREAD

**Issue by eantoranz** [michal-h21/make4ht#36]
I am working on a handbook and I am using latex to write it. I am able to produce PDFs and I also produce HTML content using make4ht (on an uptodate debian testing box).

handbook is here: https://github.com/eantoranz/conflict_book/tree/main/latex

I generate the HTML content like this:

`make4ht conflicts.tex "xhtml,3"`

The output looks more or less like what I am looking forward to:

http://www.ezconflict.com/en/conflicts.html

However, I noticed that the links that are at the top/bottom of pages are not consistent. Say, you go to Conventions.

http://www.ezconflict.com/en/conflictsse5.html#x7-50000.5

There is a link at the top that says `next`. If I click on it, I would expect to be taken to the next part, which is the `Basics` Chapter, `What is a conflict?` section http://www.ezconflict.com/en/conflictsse6.html#x11-70001.1... or at least the `Basics` chapter http://www.ezconflict.com/en/conflictsch1.html#x10-60001.... instead we get taken to the third section in `Basics` Chapter, `Terms and Acronyms` http://www.ezconflict.com/en/conflictsse8.html

Is there something I am doing wrong when generating the HTML with make4ht?

**Comment by michal-h21** [comment]
I've provided some explanation here: https://tex.stackexchange.com/a/561820/2891

TLDR: you may need to run `make4ht` multiple times when your section change. In subsequent runs, you can run `make4ht -m draft` to save time. Just keep the temporary files.

**Comment by eantoranz** [comment]
Came here to close the ticket. Thanks for taking care of that. Very kind for the help.
